cli/src/cli.rs

```diff
@@ -21,9 +21,26 @@ use structopt::StructOpt;
 #[allow(missing_docs)]
 #[derive(Debug, StructOpt)]
 pub enum Subcommand {
-	#[allow(missing_docs)]
-	#[structopt(flatten)]
-	Base(sc_cli::Subcommand),
+	/// Build a chain specification.
+	BuildSpec(sc_cli::BuildSpecCmd),
+
+	/// Validate blocks.
+	CheckBlock(sc_cli::CheckBlockCmd),
+
+	/// Export blocks.
```

Comment:
It would be nice if those comments could be moved to sc-cli instead so people don't need to duplicate them on every project.

(I don't know if this is possible!)

Reply:
Yeah, not sure. If I don't comment these lines there's still some text for them in the `--help` display, but it's for some of the options for the command, not the command itself. Weird.